Mathlib/Data/Buffer/Parser/Numeral.lean

```diff
@@ -0,0 +1,20 @@
+/-
+Copyright (c) 2020 Yakov Pechersky. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yakov Pechersky
+
+! This file was ported from Lean 3 source module data.buffer.parser.numeral
+! leanprover-community/mathlib commit 2220b0cbab795e73674b8191170b0cc68c6b54a8
+! Please do not edit these lines, except to modify the commit id
+! if you have ported upstream changes.
+
+Porting note:
+As the parsing framework has completely changed in Lean 4
+there is no point porting these files directly.
+They can be rewritten from scratch as needed, just as for tactics.
+-/
+
+-- Porting note:
+-- As the parsing framework has completely changed in Lean 4
+-- there is no point porting these files directly.
+-- They can be rewritten from scratch as needed, just as for tactics.
```

Comment:
```suggestion
```

Reply:
Is the suggestion here to just drop the comment?

Reply:
It's duplicated